### DISCUSSION TITLE
running import_phonopy_vasp.py occur an error

### DISCUSSION THREAD

**Discussion by gswylq** [anharmonic/d3q#7] [General]
Hello everyone,

I use the import_phonopy_vasp.py to convert the phonopy FC  to native qe fc format,  but an error occurs as following:
################
Force constant unit conversion factor: 0.0205817184255
Supercell size:  2 2 1
Skipped the wrong amount 0 400Traceback (most recent call last):
  File "import_phonopy_vasp.py", line 183, in <module>
    raise Exception
Exception
##########################

The  input files add to attachment.

Please give some help.

Thanks.

Qiang Li

[phonopy_vasp.zip](https://github.com/anharmonic/d3q/files/8519519/phonopy_vasp.zip)

**Comment by paulatz** [comment]
Hello Qiang Li, thank you for reporting.
There is a bug in the script that causes the change of coordinates of R vectors to only work for cubic systems. You can easily fix yourself by changing line 72 from
  rbigx =  dot(trcell, rbig)
to
  rbigx =  dot(rcell, rbig)
(i.e. change trcell to rcell on that line only)

I'm going to upload a fixed version on github later, but I also ported it to python3 as I was looking for the bug, if you prefer python2 just use the fix above.

cheers

**Comment by paulatz** [comment]
p.s. you may also have to remove the "break" statement at line 177, at least it won't hurt removing it.
